### PR TITLE
test(ivy): add root cause analysis for failing core tests (view injec…

### PR DESCRIPTION
**Pull request by marclaval**
…tor integration)

### CODE REVIEW

**Review by mhevery — APPROVED** [review]

File: packages/core/test/linker/view_injector_integration_spec.ts [hunk 1]
```diff
@@ -465,52 +467,55 @@ class TestComp {
            expect(comp.componentInstance.a).toBe('aValue');
          });
 
-      fixmeIvy('unknown').it('should support ngOnDestroy for lazy providers', () => {
-        let created = false;
-        let destroyed = false;
+      fixmeIvy('FW-888: Destroy on providers is not being called')
```
Comment (kara) [inline packages/core/test/linker/view_injector_integration_spec.ts]: This looks like a dupe of FW-848

File: packages/core/test/linker/view_injector_integration_spec.ts [hunk 2]
```diff
@@ -764,24 +769,25 @@ class TestComp {
...
-                                    .configureTestingModule({
-                                      declarations: [MyComp],
-                                      providers: [{provide: 'someToken', useValue: 'someValue'}]
-                                    })
-                                    .createComponent(MyComp);
-
-            expect(compFixture.componentInstance.vc.parentInjector.get('someToken'))
-                .toBe('someValue');
-          });
+      fixmeIvy('FW-892: Exception thrown when getting VCRef.parentInjector on the root view')
```
Comment (kara) [inline packages/core/test/linker/view_injector_integration_spec.ts]: This looks like a dupe of FW-638

File: packages/core/test/linker/view_injector_integration_spec.ts [hunk 3]
```diff
@@ -988,7 +993,7 @@ class TestComp {
         expect(el.children[0].injector.get(SimpleDirective).value.changeDetectorRef).toEqual(cdRef);
       });
 
-      fixmeIvy('unknown').it('should cache pure pipes', () => {
+      fixmeIvy('FW-896: Pure pipes are not properly cached').it('should cache pure pipes', () => {
```
Comment (kara) [inline packages/core/test/linker/view_injector_integration_spec.ts]: This looks like a dupe of FW-821

**Review by kara — DISMISSED** [review]

File: packages/core/test/linker/view_injector_integration_spec.ts (hunk 1 above)
Comment (marclaval) [reply packages/core/test/linker/view_injector_integration_spec.ts]: Indeed

File: packages/core/test/linker/view_injector_integration_spec.ts (hunk 2 above)
Comment (marclaval) [reply packages/core/test/linker/view_injector_integration_spec.ts]: It is

File: packages/core/test/linker/view_injector_integration_spec.ts (hunk 3 above)
Comment (marclaval) [reply packages/core/test/linker/view_injector_integration_spec.ts]: 👍

**Comment by AndrewKushnir** [comment]
@marclaval could you please rebase from the latest master and resolve conflict (and add "merge" label back)? Thank you.

**Comment by marclaval** [comment]
No longer relevant, it was merged with https://github.com/angular/angular/pull/27912

**Pull request closed without merging**